java/bench/core/src/java/org/apache/orc/bench/core/CompressionKind.java

```diff
@@ -36,8 +36,8 @@
   SNAPPY("snappy"),
   ZSTD("zstd");
 
-  CompressionKind(String extendsion) {
```

Comment:
Nice catch. :)

Reply:
I found a few, but most of them were scanned with tools. :blush: